README.md

```diff
@@ -6,14 +6,53 @@
 $ bower install gaia-components/gaia-button
 ```
 
+Then include folowing files in HTML
+
+```html
+<link href="bower_components/gaia-icons/gaia-icons.css"></link>
+<script src="bower_components/gaia-list/gaia-button.js"></script>
+```
+
+You don't have to include `gaia-icon.css` if you do not use icon in your button.
+
+
 ## Examples
 
 - [Example](http://gaia-components.github.io/gaia-button/)
 
+
 ## Usage
 
+Button that contain text need to include `data-l10n-id` attribute for localization.
+
 ```html
-<gaia-button>normal button<gaia-button>
+<gaia-button><span data-l10n-id="normal_button"/><gaia-button>
+```
+
+### With icon
+
+* {Start icon} button text
+
+Button can show with icon at left
+
+```
+<button><span data-icon="back-light" data-l10n-id="back"/></button>
```

Comment:
Thank its typos for `gaia-button`.

It looks strange to me to repeatedly add `data-l10n-id="back"` to a button, and it might cause extra parsing/replacing in l20n?